src/helpers/amoc_xmpp_handlers.erl

```diff
@@ -15,6 +15,11 @@
 %% Helpers
 -export([ttd/2]).
 
+-required_variable(#{name => just_log_errors_in_handlers, default_value => false,
+                     description => "setting this flag prevents crashing on failures"
+                                    " in xmpp handlers (predicates or actions)",
+                     verification => fun erlang:is_boolean/1}).
```

Comment:
This should work:
```suggestion
                     verification => fun is_boolean/1}).
```

Reply:
I'm afraid it's not allowed to use this format of function reference in module attributes, only `fun M:F/A` is allowed (i.e. only exported function).

Proposed change results in compilation error:
```
===> Compiling src/helpers/amoc_xmpp_handlers.erl failed
src/helpers/amoc_xmpp_handlers.erl:18:20: bad attribute
```